test/naive_test.py

```diff
@@ -23,27 +23,31 @@
 import sys
 import time
 import traceback
+import os
+import argparse
 
 from utils import is_experiment_done, fetch_nni_log_path, read_last_line, remove_files, setup_experiment
 from utils import GREEN, RED, CLEAR, EXPERIMENT_URL
 
```

Comment:
naive_test.py do not need to support remote machine

Reply:
removed remote mode in naive_test.py